packages/marketplace/src/selector/client.ts

```diff
@@ -6,11 +6,26 @@ import { WebComponentConfigResult } from '@/services/web-component'
 import { InstalledAppsState } from '@/reducers/installed-apps'
 import { AppSummaryModel } from '@reapit/foundations-ts-definitions'
 import { ClientAppSummaryState } from '@/reducers/client/app-summary'
+import { selectLoginIdentity } from '@/selector/auth'
+import { COGNITO_GROUP_DEVELOPER_EDITION } from '@/constants/api'
 
 export const selectClientId = (state: ReduxState) => {
   return state?.auth?.loginSession?.loginIdentity?.clientId || ''
 }
 
+/**
+ * Need get developer id to filter apps list if this user belong to
+ * AgencyCloudDeveloperEdition group, if not just return null
+ * refer to this ticket https://github.com/reapit/foundations/issues/1848
+ */
+export const selectDeveloperId = (state: ReduxState) => {
```

Comment:
Naming seems ambiguous to me even with comment on the func. Imagine we will need to selectDeveloperId in the future without this logic

Reply:
@phmngocnghia's comment makes sense, and should add return value type too.
It may cause confusion when we splitting to Client/Dev portal next week

Reply:
`selectDeveloperEditionId` as a name maybe?